src/components/LanguagePicker/LanguagePickerList.tsx

```diff
@@ -0,0 +1,87 @@
+import React, {useState} from 'react'
+import {FlatList, StyleSheet, View} from 'react-native'
+
+import {LanguagePickerListItem} from './LanguagePickerListItem'
+import {LanguagePickerListSearchInput} from './LanguagePickerListSearchInput'
+
+export function LanguagePickerList({
+  data,
+  idSelected,
+  setIdSelected,
+  placeholderText = 'Search',
+  withSearch = true,
+  EmptyListComponent,
+  defaultId = null,
+}: Props) {
+  const [itemToSearch, setItemToSearch] = useState<string>('')
+
+  const normalizedText = normalize(itemToSearch)
+  const dataFiltered = data.filter((item) => normalize(item.text).startsWith(normalizedText))
+  const emptyList = itemToSearch && dataFiltered.length === 0 && EmptyListComponent ? <EmptyListComponent /> : null
+
+  const onItemPressed = (item) => (item.id === idSelected ? setIdSelected(defaultId) : setIdSelected(item.id))
+
+  return (
+    <View style={styles.container}>
+      {withSearch && (
```

Comment:
reserve `withX` for higher order components

Reply:
Component removed. I will apply next time thanks!